### ISSUE TITLE
Error, When i run cloud image. Please help me !

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [ghik/kubernetes-the-harder-way#8]
Hi Ghik, Thank for your repo that is a good resource. But when i run i have a error like 

When i run 

```
sudo qemu-system-aarch64 \
    -nographic \
    -machine virt,accel=hvf,highmem=on \
    -cpu host \
    -smp 2 \
    -m 2G \
    -bios /opt/homebrew/share/qemu/edk2-aarch64-code.fd \
    -nic vmnet-shared \
    -hda ubuntu0.img
```
<img width="897" alt="image" src="https://github.com/ghik/kubernetes-the-harder-way/assets/22360388/2f5d9a75-9a14-482d-b5cc-401dc21bd743">

I hope you can help me to fix this error

**Comment by ghik** [comment]
Hi @NguyenThaiHoc1, please see the discussion in #6

**Comment by ghik** [comment]
Closing as a duplicate of #6 (which seems fixed in QEMU 8.2.1)
